### PR TITLE
[ADD] open_academy: Creation of Courses Menu T#59081

### PR DESCRIPTION
**Pull request by JesusValdez96**
Add menu entries to access courses of the open_academy module
Link to exercise: https://www.odoo.com/documentation/15.0/developer/howtos/backend.html#actions-and-menus

Add __pycache__ and .vscode configuration to gitignore

Fix name of the module in manifest that was changed due to a mistake in the last commit

### CODE REVIEW

**Comment by JesusValdez96** [comment]
Please review @andreaarce, @CarmenMiranda :smiley:

File: open_academy/__manifest__.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
     "data": [
         "security/ir.model.access.csv",
+        "views/course_menu.xml"
```
Comment (CarmenMiranda) [inline open_academy/__manifest__.py:20]: Missing comma at the end of the line

**Review by CarmenMiranda — CHANGES_REQUESTED** [review]
Just a little comment 🙈

**Comment by JesusValdez96** [comment]
I forgot that :see_no_evil: 
Please review again @CarmenMiranda, @andreaarce

**Review by CarmenMiranda — APPROVED** [review]
@andreaarce Technically LGTM 👍🏼 

@JesusValdez96 Just remove the Draft in the title 🙈

**Pull request merged**